scripts/VisionSuite/utils.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env python
+
+""" 
+utils.py
+-----
+
+Adds visualization and boundery setting helper funtions for visual suite
+
+"""
+
+import cv2, math
+import numpy as np
+
+class ColorFilter(object):
+    """ 
+    ColorFilter is a Python object that can visualize and set color filter
+    bounding values on an image
+    """
+    def __init__(self, img, bounds={"upperBGR":np.array([255,255,255]),"lowerBGR":np.array([0,0,0])}):
+        """ Initialize the ball tracker """   
+
+        #set initial boundry values
+        self.lower = bounds["lowerBGR"]
+        self.upper = bounds["upperBGR"]
+        
+        #create images     
+        self.IMG = img 
+
+    def createBounds(self):
```

Comment:
If you're interested, I wrote a generalization of the slider generation code that makes this a *bit* more compact and scalable than writing out individual callbacks for 12 slider variables, but honestly, it'd probably be a distraction if you've got functionality to build out still.